### ISSUE TITLE
v2.2 is behaving strangely for me - Any thoughts?

### ISSUE THREAD

**Issue by korlaxxalrok** [jaroslawhartman/TimeZones-Alfred#10]
Hi @jaroslawhartman - Thanks for taking on the task of keeping this workflow alive and updated. It is one of my favorites.

Here is a screen cap of me trying to invoke `tz` on v2.2. Any idea what might be causing this?

![image](https://user-images.githubusercontent.com/3670389/50012606-0be79c80-ff8d-11e8-9849-0b60db27f208.png)

**Comment by jaroslawhartman** [comment]
Indeed, it does, at many aspects... Wondering if you could post here your configuration file (choose _Edit file..._:

![image](https://user-images.githubusercontent.com/964833/50056906-93870400-0163-11e9-8c35-1ab4a565fb8d.png)

If this is looking good, all I can suggest is to:

- delete your configuration file: `rm "$HOME/Library/Application Support/Alfred 3/Workflow Data/carlosnz.timezones/timezones.txt"`
- add your cities
- remove and reinstall the workflow

Let's see if it will help,
Jarek

**Comment by jaroslawhartman** [comment]
Thank you Brian.
